### ISSUE TITLE
unexpected error: `malformed request: workflows must be specified with a 'metadata.name=...' field selector`

### ISSUE THREAD

**Issue by alexec** [argoproj/argo#3625]
```
nfoService grpc.start_time="2020-07-28T19:52:15-07:00" grpc.time_ms=0.046 span.kind=server system=grpc
argo-server | time="2020-07-28T19:52:15-07:00" level=error msg="finished streaming call with code Unknown" error="malformed request: workflows must be specified with a 'metadata.name=...' field selector; unable to parse: incorrect prefix: selector is not formatted correctly" grpc.code=Unknown grpc.method=WatchWorkflows grpc.service=workflow.WorkflowService grpc.start_time="2020-07-28T19:52:15-07:00" grpc.time_ms=0.097 span.kind=server system=grpc

```

**Comment by simster7** [comment]
Could you provide more details as to how this happened? Likely this request was malformed

**Comment by alexec** [comment]
This is loading the workflow list page in the UI. No changes to workflows appear in the UI. It is broken.

**Comment by simster7** [comment]
Looking into this now

**Comment by alexec** [comment]
Fixed and available in v2.10.0-rc5.

**Comment by alexec** [comment]
Available in [v2.9.5](https://github.com/argoproj/argo/releases/tag/v2.9.5)
